core/webhooks/webhooks.go

```diff
@@ -1,17 +1,11 @@
 package webhooks
 
 import (
-	"bytes"
-	"encoding/json"
-	"net/http"
-	"time"
+  "time"
```

Comment:
https://pkg.go.dev/golang.org/x/tools/cmd/goimports?utm_source=godoc

Might be a whitespace thing that goimports is complaining about?